packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -45,6 +45,19 @@ static inline CGFloat GetMinDistanceBetweenRectVerticesAndPoint(NSRect rect, NSP
 	);
 }
 
+static NSView *GetFirstKeyViewWithin(NSView *parentView)
+{
+	for (NSView *view in [parentView subviews]) {
+		if ([view canBecomeKeyView]) {
+			return view;
+		}
+		
+		NSView *match = GetFirstKeyViewWithin(view);
+		if (match) return match;
```

Comment:
I think there's some Obj-c guidelines around this somewhere but we should use curly braces even for 1 line conditional actions. I think the main benefit is debugging and handling merge conflicts - we've been burnt before with merges that hit this